pyroengine/inference/configManager.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```

Comment:
for a python package, it would be better to have snake case naming of file :pray: